share/picongpu/examples/FieldAbsorberTest/include/picongpu/param/fieldSolver.param

```diff
@@ -37,6 +37,8 @@
 #include "picongpu/fields/MaxwellSolver/Solvers.def"
 #include "picongpu/fields/currentInterpolation/CurrentInterpolation.def"
 
+#include <boost/preprocessor/punctuation/comma.hpp>
+
```

Comment:
Sorry I didn't realize this before, when this solution was suggested. This adds a dependency on Boost.propocessor library, that we do not currently have. I do not think it's worth it honestly. @psychocoderHPC what are your thoughts? If we decide to add it, should update cmakelists.txt and documentation as well. I think a more practical solution would be to just make another macro that decides on Yee / AOFDTD and express a combination via 2 separate macros.

Reply:
PMacc already depends on boost PP, I am fine with both solutions.

Reply:
@sbastrakov How about you? Do you still require changes?

Reply:
Ah, of course. I only looked at PIConGPU's cmakelists which does not. Okay, then I'm lifting my concern.